packages/forma-36-website/src/content/guidelines/contributing.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: 'Contributing to Forma 36'
+slug: '/guidelines/contributing/'
+---
+
+Forma 36 is an open-source design system by Contentful and we welcome contributions from our community.
+If we have an idea that could help other users in this page we will teach you how to do it.
+
+## Table of contents
+
+- [Before contributing](#before-contributing)
+  - [Check if your idea is already present](#check-if-your-idea-is-already-present)
+  - [Could an existing component be extended to realize your idea?](#could-an-existing-component-be-extended-to-realize-your-idea)
+- [Contribution Process](#contribution-process)
+  - [Step 1: Open a Github issue](#step-1-open-a-github-issue)
+  - [Step 2: Proposal Review](#step-2-proposal-review)
+  - [Step 3: Open a Pull Request](#step-3-open-a-pull-request)
+  - [Step 4: Code & Design Review](#step-4-code--design-review)
+  - [Step 5: Merge the changes to the main branch](#step-5-merge-the-changes-to-the-main-branch)
+- [Bugs & Light contributions](#bugs--light-contributions)
+  - [Examples of light contributions](#examples-of-light-contributions)
+
+## Before contributing
+
+### Check if your idea is already present
+
+Before writing a proposal or any code, we ask you to go around this website or our Github repo and
+check if your idea is not yet implemented.
```

Comment:
Maybe this can be more specific? "Search this website" or something. It can be easy to lose track of what already exists, especially on GitHub.

Reply:
but I'm saying "look around the website and Github"

I think I didn't understand the suggestion, could you re-write this part so I can see what you have in mind? 🙏

Reply:
If you're completely new you probably won't know what to look for or where, so I think "we ask you to go around this website or our Github repo" can be more actionable. For instance it could be something like "Use the search bar to the left" (or something better 😅 )

Reply:
I would prefer to keep both suggestions, the website and github repo (with a link if possible), because there will be users that would prefer checking the documentation and users that would prefer to go into the codebase and check it by themselves.

Reply:
maybe add a direct link in the text to the github repo.
has anyone access to our community slack? we could also ask them to go into the community slack forma-36 channel and ask questions there if they are unsure if their idea is yet implemented?